### PR TITLE
add Eclair #1339

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

File: _posts/en/newsletters/2020-03-25-newsletter.md [hunk 1]
```diff
@@ -40,7 +40,12 @@ FIXME:bitschmidty
 [libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Eclair #1339][] Harden requirements on htlc-minimum-msat FIXME:ajonas
+- [Eclair #1339][] prevents users from setting their htlc-minimum
+  amount to 0 milli-satoshis. The new minimum is 1 milli-satoshi. This is
+  in response to [LND #3697][], which lowered its previous default from 1,000
+  msat to 0 msats to allow sub-satoshi payments as covered in
+  [Newsletter #88][newsletter lnd 3697]. With this change, Eclair will no
```
Comment (harding) [inline _posts/en/newsletters/2020-03-25-newsletter.md]: s/88/77/

**Review by harding — CHANGES_REQUESTED** [review]
Typo fix needed.  I'm not fond of the "this is in response" phrasing as I feel it implies LND did something wrong.  AFAIU, their default policy wouldn't allow the creation of 0-value HTLCs as they still have a 1 msat minimum for actual routing; it's just that they now signal that their nodes may try to send future payments of 0 msats, which made the Eclair devs aware that they weren't prepared for that possibility.  Maybe something like s/in response/may be influenced by/ to give it a more neutral or positive tone.

File: _posts/en/newsletters/2020-03-25-newsletter.md (hunk 1 above)
Comment (adamjonas) [reply _posts/en/newsletters/2020-03-25-newsletter.md]: Sloppy. Sorry.

**Comment by adamjonas** [comment]
Corrected typo and made change on tone for LND #3697. Thanks!

**Comment by harding** [comment]
Thank you!

**Pull request merged**